src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -478,4 +478,12 @@ protected function isHttpException(Exception $e)
     {
         return $e instanceof HttpException;
     }
+
+    /**
+     * @return \Illuminate\Config\Repository|mixed
+     */
+    protected function isDebugMode()
+    {
```

Comment:
`inDebugMode` or `debugEnabled` would make more sense to me.

Reply:
ok, i will change to inDebugMode